### PR TITLE
feat : landing page

### PR DESCRIPTION
**Pull request by Adarsh-Patel-Dev**
Files to Check: 
 - LandingPage
 - Components
    -Asidebar
    -Card
    -Footer
    -Navigation
    
    
image

image

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,18 +1,20 @@
...
 import React from 'react';
-import { Route, Routes } from "react-router-dom";
-import Navbar from "./components/Navibar/Navbar";
-import HomePage from "./pages/LandingPage/HomePage/HomePage";
+import { AsideBar } from "./components/Asidebar/Asidebar";
+import { Hero } from "./components/HeroSection/Hero";
+import {LandingPage} from "./pages/LandingPage/LandingPage";
+import { Footer } from "./components/Footer/Footer";
+import { CardHorizontal } from "./components/Card/CardHorizontal";
+
```
Comment (sri-rishi) [inline src/App.js:9]: Suggestion: you can import all files in separate index.js in their folder and use it from there.

File: src/backend/db/categories.js [hunk 2]
```diff
@@ -4,33 +4,44 @@ import { v4 as uuid } from "uuid";
  * Category Database can be added here.
  * You can add category of your wish with different attributes
  * */
+/*
+   <iframe width="697" height="392" src="https://www.youtube.com/embed/PKZABq7Llzs" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+*/
```
Comment (MayankKumar10) [inline src/backend/db/categories.js:9]: suggestion: remove the comment from here

File: src/components/Card/CardHorizontal.jsx [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+import './card.css'
+import { BiLike } from "react-icons/bi";
+import { MdOutlineWatchLater, MdPlaylistPlay } from "react-icons/md";
+
+function CardHorizontal() {
+  return (
+    <div className="card">
+           
+            <div className="card--image">
+                <img src='https://i.ytimg.com/vi/Ir7dFd9Oflw/hq720.jpg?sqp=-oaymwEcCNAFEJQDSFXyq4qpAw4IARUAAIhCGAFwAcABBg==&rs=AOn4CLBTaMoMZZfxlblwsVcy_XpOxnV17Q' className="img-fluid" alt="Plant-Image" width="400px" height="20px"/>
```
Comment (sri-rishi) [inline src/components/Card/CardHorizontal.jsx:12]: Suggestion: ignore inline styling

File: src/components/Footer/Footer.jsx [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+           <p>Connect with me</p>
+          <ul className='social-media-icons'>
+            <li><FaLinkedinIn/></li>  
+            <li><FaTwitter/></li>  
+            <li><FaGithub/></li>  
+            <li><FaInstagram/></li>  
+          </ul>
+            <span>Made with <FaHeart/> by Adarsh </span>
+         </div>
+         <a href='#' class="btn btn--primary btn--float">
```
Comment (sri-rishi) [inline src/components/Footer/Footer.jsx:22]: Suggestion: I think you forgot to change class to className here

File: src/backend/db/categories.js [hunk 5]
```diff
@@ -4,33 +4,44 @@ import { v4 as uuid } from "uuid";
...
+    iframe: "" ,
+    image: "https://i.ytimg.com/an_webp/PKZABq7Llzs/mqdefault_6s.webp?du=3000&sqp=CMukipMG&rs=AOn4CLDU_4QTTt2cDXvPY1uuqP6sy3-H-g",
     description:
       "Computer programming is the process of designing and building an executable computer program to accomplish a specific computing",
   },
   {
     _id: uuid(),
     categoryName: "indoor",
-    image: "",
+    url:"https://youtu.be/kWr8SvbUGMo",
```
Comment (MayankKumar10) [inline src/backend/db/categories.js:24]: Suggestion: You don't need to write the whole URL just id of that video would also be fine, and store youtube in some variable.

File: src/components/HeroSection/Hero.jsx [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+        <p className="hero-section-para">Today</p>
+        <p className="hero-section-para">will help you Tomorrow</p>
+     <div className="cta-btns">
+       {/* <NavLink to='/products'> */}
+       <button className="primary-cta" >Watch Now <FaArrowRight style={{fontSize: "18px"}}/></button>
+       {/* </NavLink> */}
+    </div>
+</div>
+<div className="img-hero">
+    <img src='https://images.unsplash.com/photo-1538506392157-218c11f95129?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1055&q=80'/>
```
Comment (sri-rishi) [inline src/components/HeroSection/Hero.jsx:21]: Suggestion: provide img tag a `alt` attribute

File: src/components/Asidebar/asidebar.css [hunk 7]
```diff
@@ -0,0 +1,45 @@
+
+.aside-container {
+    padding: 0;
```
Comment (MayankKumar10) [inline src/components/Asidebar/asidebar.css:3]: Suggestion: define this in the root or body element so it will inherit by default.

**Review by sri-rishi — APPROVED** [review]
Great work, just some suggestion which I think can be done, feel free to merge

File: src/components/Asidebar/asidebar.css [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+    flex-direction: column;
+}
+
+
+
+.aside-item{
+    width: auto;
+    display: flex;
+    flex-direction: column;
```
Comment (MayankKumar10) [inline src/components/Asidebar/asidebar.css:30]: Suggestion: Try to make helper class for flexbox so you can use it anywhere

**Review by MayankKumar10 — APPROVED** [review]
Just Look into these Suggestions everything else looks great Nice UI you can merge it.

**Pull request merged**